### PR TITLE
Prepare for on boarding

### PR DESCRIPTION
**Pull request by HarelM**
I believe this solves most of the missing checkboxes:
- https://github.com/maplibre/maplibre/issues/352

Let me know if there's anything else missing.
Other configuration can be done after the repo has been migrated I believe.

### CODE REVIEW

**Comment by birkskyum** [comment]
I was thinking about having the small [docs / landing page](https://maputnik.github.io/) in this repo as well, and push that to maplibre.org/maputnik. What do you think of that?

**Review by birkskyum — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2015 Lukas Martinelli
+Copyright (c) 2024 MapLibre contributors
```
Comment (HarelM) [inline LICENSE:4]: Should this be only Maplibre or keep it with Lukas as it is now?

Comment (birkskyum) [reply LICENSE:4]: Think just maplibre is fine in this case

**Comment by HarelM** [comment]
I was thinking that `maplibre.org/maputnik` will be the editor itself and add some info to the maplibre.org site somewhere, like we do for Martin maybe or in another page.
Maputnik is pretty straight forward, I don't see a reason to add a "full" landing page for it.
But I'm good with everything that we decide.

**Comment by HarelM** [comment]
OK, pushing the button...

File: LICENSE (hunk 1 above)
Comment (wipfli) [reply LICENSE:3]: In all other repos we kept the old copyright holder and just added MapLibre. I think we should do the same here...

Comment (nyurik) [reply LICENSE:3]: we can never remove existing copyrights

File: README.md [hunk 2]
```diff
@@ -102,60 +101,10 @@ npm run cy:open
...
+Read more about the MapLibre Sponsorship Program at https://maplibre.org/sponsors/.
 
 ## License
 
-Maputnik is [licensed under MIT](LICENSE) and is Copyright (c) Lukas Martinelli and contributors.
-
-**Disclaimer** This is an independent style editor.
-As contributor please take extra care of not violating any Mapbox trademarks. Do not get inspired by Mapbox Studio and make your own decisions for a good style editor.
+Maputnik is [licensed under MIT](LICENSE) and is Copyright (c) Lukas Martinelli and Maplibre contributors.
+As contributor please take extra care of not violating any Mapbox trademarks. Do not get inspired by other map studios and make your own decisions for a good style editor.
```
Comment (nyurik) [inline README.md:110]: inspiration is a bit vague - i am not sure about the legal aspects of that. We must never copy code for sure

Comment (nyurik) [reply README.md:110]: oh, my appologies, this was in the original text - lets keep it then

**Comment by HarelM** [comment]
This was merged, simply create a PR with the changes you want and let's merge them.

**Pull request merged**